packages/flutter/lib/src/material/segmented_button_theme.dart

```diff
@@ -0,0 +1,131 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'button_style.dart';
+import 'theme.dart';
+
+// Examples can assume:
+// late BuildContext context;
+
+/// Defines the color and border properties of [SegmentedButton] widgets.
+///
+/// Used by [SegmentedButtonTheme] to control the color and border properties
+/// of toggle buttons in a widget subtree.
```

Comment:
Grep for "Toggle" and especially "ToggleButtons" in all of the files in this PR.